### ISSUE TITLE
Container snapshots for fast database reloads

### ISSUE THREAD

**Issue by mortona42** [lando/lando#3577]
Long ago, back when vagrant was our favorite local dev environment, I used this sahara plugin to take a snapshot of the box that is quicker to reload: https://github.com/jedi4ever/sahara.

This is fantastic for testing migrations or other operations that modify the database, which you need to revert to test again.

I'm a docker n00b so I'm not sure what the corresponding option is here. If there's a fast way to save and load a copy of the container, we could set up commands to switch it out.

**Comment by GoonerW** [comment]
You can always run lando db-export and lando db-import to backup and restore the database.

**Comment by mortona42** [comment]
Yep, that's what I'm doing now but it can take a while for a large database. I'm looking for something faster.

**Comment by GoonerW** [comment]
Fair enough

**Comment by pirog** [comment]
fwiw @mortona42 we are planning on a "lando save" command in Lando 4 that, under the hood, will take container snapshots
